### DISCUSSION TITLE
Support different languages

### DISCUSSION THREAD

**Discussion by team-deepgram** [deepgram/community#830] [General help] [answered]
What parameter I need to send in the API request to support multiple languages at once?

> [Posted by farahxr](https://discord.com/channels/1108042150941294664/1255177284697194527/1255177284697194527)

**Comment by team-deepgram** [comment, marked as answer]
Hi Farah, we don't have language switching yet. We're working on it though



> [Posted by lukeocodes](https://discord.com/channels/1108042150941294664/1255177284697194527/1255183121477795910)

**Comment by EdwinCalvo** [comment]
Now it's available with the language='multi' option:
https://developers.deepgram.com/docs/multilingual-code-switching
